### ISSUE TITLE
JAWS detects all Chat windows as RichEdit Control

### ISSUE THREAD

**Issue by georgehazan** [miranda-ng/miranda-ng#360]
**Reported by winman3000 on 12 Jan 2015 18:01 UTC**
I am a blind Person and I am using JAWS as my Screen Reader. Since I updated Miranda NG, JAWS detects all Chat Edit Fields as RichEdit Control. The Window Title is also RichEdit Control.

**Comment by georgehazan** [comment]
**Comment by ghazan on 12 Jan 2015 19:04 UTC**
yes, all chat edit fields were RichEdit for years.
what is the question?

**Comment by georgehazan** [comment]
**Comment by winman3000 on 12 Jan 2015 19:10 UTC**
Replying to [ghazan](comment:1):
> yes, all chat edit fields were RichEdit for years.
> what is the question?

In Miranda NG 0.95.2, JAWS detects all Fields as Edit Field and the Window Title is detected correctly. Since I've updated to 0.95.3, JAWS says "RichEdit Control" and if I want to know the Window Title, wich should be the Name of the Chat Person, JAWS Says "RichEdit Control". This is only in Windows 8.1, not in Windows 7.

**Comment by georgehazan** [comment]
**Comment by ghazan on 13 Jan 2015 15:21 UTC**
then you need to issue a bugreport on JAWS, not here.
msftedit was introduced in Windows XP, so they had 12 years to prepare their program

**Comment by georgehazan** [comment]
**Comment by winman3000 on 13 Jan 2015 15:30 UTC**
I am wondering, why it is working with 0.95.2? Is anything changed between 0.95.2 and 0.95.3 for Edit Fields? If not, I'll search the reason.
